routing/routes.go

```diff
@@ -793,6 +793,27 @@ func NewRoutes() (r *Routes) {
 			handlers.NamespaceHealth,
 			true,
 		},
+		// swagger:route GET /namespaces/{namespace}/validations namespaces namespaceValidations
+		// ---
+		// Get validations for all objects in the given namespace
+		//
+		//     Produces:
+		//     - application/json
+		//
+		//     Schemes: http, https
+		//
+		// responses:
+		//      200: namespaceValidationsResponse
+		//      400: badRequestError
+		//      500: internalError
+		//
+		{
+			"NamespaceHealth",
```

Comment:
Wrong name in the ID. "NamespaceHealth" -> "NamespaceValidationsSummary" or something similar.